app.css

```diff
@@ -1,34 +1,80 @@
-body {
+html  {
+  height: 100%;
+  width: 100%;
   font-size: 1rem;
   font-family: 'Cabin', sans-serif;
+  text-align: center;
+}
+
+#title {
```

Comment:
This positioning looks _really_ finnicky. What are you trying to accomplish here?